components/Hero.vue

```diff
@@ -58,16 +52,21 @@ export default {
 
   .title {
     color: $dark-blue;
+    line-height: 1em;
     .gradient {
-      background: linear-gradient(
-        90deg,
-        $aquamarine-1 0%,
-        $aquamarine-1 43%,
-        $green-2 100%
-      );
+      background: linear-gradient(90deg, $aquamarine-1 43%, $green-2 100%);
       background-clip: text;
       -webkit-text-fill-color: transparent;
     }
+    &::after {
```

Comment:
I think it is better if you include the dot inside a span to add a different style. Or make the container bigger so it fits on one line
<img width="596" alt="Captura de pantalla 2021-04-14 a las 16 18 31" src="https://user-images.githubusercontent.com/16032491/114726213-76679d80-9d3d-11eb-9560-bd17c9362c5a.png">

Reply:
Problem is that none of the imported fonts uses a square dot character.

Reply:
Let's wait for the Avenir font to be in place and then adjust so it falls in the same line.